### PR TITLE
add tests for adding md5 and organism id in uniparc

### PR DESCRIPTION
**Pull request by ahmadshadab**

### CODE REVIEW

File: support-data-rest/src/test/java/org/uniprot/api/support/data/configure/service/UniParcConfigureServiceTest.java [hunk 1]
```diff
@@ -38,7 +38,7 @@ void getSearchItems() {
         UniParcConfigureService service = new UniParcConfigureService();
         List<AdvancedSearchTerm> result = service.getSearchItems(CONTEXT_PATH);
         assertNotNull(result);
-        assertEquals(14, result.size());
+        assertEquals(15, result.size());
```
Comment (supun-ebi) [inline support-data-rest/src/test/java/org/uniprot/api/support/data/configure/service/UniParcConfigureServiceTest.java:41]: what made this change? I dont see any change in a main (non test) class.

**Review by supun-ebi — APPROVED** [review]

**Review by supun-ebi — COMMENTED** [review]
comment

File: support-data-rest/src/test/java/org/uniprot/api/support/data/configure/service/UniParcConfigureServiceTest.java (hunk 1 above)
Comment (ahmadshadab) [reply support-data-rest/src/test/java/org/uniprot/api/support/data/configure/service/UniParcConfigureServiceTest.java:41]: It is coming from store module where we keep the configs.

**Review by supun-ebi — APPROVED** [review]

**Comment by ahmadshadab** [comment]
Hi @LeonardoGonzales  @supun-ebi - could you please finish this review?

**Review by supun-ebi — APPROVED** [review]

**Review by LeonardoGonzales — APPROVED** [review]

**Pull request merged**